src/features/device-heartbeat/index.ts

```diff
@@ -306,14 +305,21 @@ export class DeviceOnlineStateManager extends EventEmitter<{
 									DeviceOnlineStates.Offline,
 									API_HEARTBEAT_STATE_TIMEOUT_SECONDS, // put the device into a timeout state if it misses it's scheduled heartbeat window... then mark as offline
 								),
-								this.updateDeviceModel(deviceId, DeviceOnlineStates.Timeout),
+								this.updateDeviceModel(
+									deviceId,
+									DeviceOnlineStates.Timeout,
+								).catch(() => {
+									// Ignore errors
+								}),
 							]);
 							break;
 						case DeviceOnlineStates.Offline:
 							await this.updateDeviceModel(
 								deviceId,
 								DeviceOnlineStates.Offline,
-							);
+							).catch(() => {
+								// Ignore errors
```

Comment:
TBH I'm not sure why those two calls have always being silencing those errors :thinking:
The only thing I can imagine is so that `deleteMessageAsync()` gets called so that the queue gets cleaned-up rather than keep indefinitely grow, but I'm not convinced about that either.

Rsmq though does mention that received but not deleted messages will be re-received after the RSMQ_READ_TIMEOUT timeout, so I'm not sure whether that's the real reason :thinking: , so I just did the least intrusive change for this PR.

Reply:
Could it be so that we don't keep re-receiving the message and re-running the PATCH forever, in case that the PATCHes fail b/c of DB load, resulting even more load to the DB?